ArduSub/Parameters.h

```diff
@@ -208,7 +208,7 @@ class Parameters {
         k_param_throttle_gain,
         k_param_cam_tilt_center, // deprecated
         k_param_frame_configuration,
-
+        k_param_surface_sensorless_thrust,
```

Comment:
sorry I'm going to be pedantic but I wonder if we could be consistent about whether we use  "nobaro" vs "sensorless"?  It's one of those golden rules where we try not to use synonyms.  This helps both with searchability (e.g. searches for "nobaro" will turn up both) and it helps with non-native english speakers who may not know about the synonym.